chains/v2/chains_dev.json

```diff
@@ -374,7 +382,7 @@
             },
             {
                 "name": "Sub.ID",
-                "account": "https://sub.id/#/{address}"
+                "account": "https://sub.id/{address}"
```

Comment:
also need to change Sub.ID for karura on 643 line

Reply:
done